### DISCUSSION TITLE
Build CefViewCore for windows 8.1

### DISCUSSION THREAD

**Discussion by rohitrajsharma16171** [CefView/CefViewCore#35] [Q&A]
Can I build CefViewCore and QCefView for Windows 8.1? If yes, then which version of CefViewCore and QCefView is compatible with Windows 8.1 (both x64 and x86)?

I am using Visual Studio 14 and windows 8.1 SDK, MSVC compiler (19.0.24215.1) with vcpkg and cmake for build.
currently I am using Cef_SDK_VER = 89.0.12+g2b76680+chromium-89.0.4389.90 but facing issues

**Comment by imofficialvivek** [comment]


**Reply by tishion** [reply]
CEF 109 is the last version supports Windows 7/8/8.1

**Reply by imofficialvivek** [reply]
> CEF 109 is the last version supports Windows 7/8/8.1

Okay Thanks

**Comment by tishion** [comment]
> Can I build CefViewCore and QCefView for Windows 8.1? If yes, then which version of CefViewCore and QCefView is compatible with Windows 8.1 (both x64 and x86)?
> 
> I am using Visual Studio 14 and windows 8.1 SDK, MSVC compiler (19.0.24215.1) with vcpkg and cmake for build. currently I am using Cef_SDK_VER = 89.0.12+g2b76680+chromium-89.0.4389.90 but facing issues

some new features of QCefView broken the compatible with earlier cef versions. 

Fixed already, please have a try.
